README.md

```diff
@@ -149,7 +149,34 @@ translations_list = [
     ('de_DE', 'German'),
 ]
 
-content_prefix = "ecosystem/finance"
+docs_url_prefix = "ecosystem/finance"
+```
+
+## Enable Previous Releases
+
+This feature allows you to link to previous versions of the docs in the left sidebar.
+
+First, start additionally deploying your docs to `<project-prefix>/stable/<version>/`, e.g. `/ecosystem/finance/stable/0.5/index.html`. See https://github.com/Qiskit/qiskit-experiments/blob/227867937a08075092cd11756214bee3fb1d4d3d/tools/deploy_documentation.sh#L38-L39 for an example.
```

Comment:
I think I'd prefer an inline `rclone` example here that works with your `ecosystem/finance` example rather than a link to a different repo's code.

Reply:
I was thinking it's better to link to a repo because it's a more complete example, such as showing how to determine what the version is. It seems like every project is using the same script to deploy docs, so I think this should be fine?